teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -1340,6 +1340,10 @@ private class BuildImpl(bean: BuildBean,
         instance.service.cancelBuild(id.stringId, request)
     }
 
+    override fun getResultingParameters(): List<Parameter> {
+        return instance.service.resultingProperties(id.stringId).property?.map { ParameterImpl(it) } ?: parameters
```

Comment:
Hmm, returning `parameters` means that client will get incorrect results because they won't include default parameters added by TeamCity. If we cannot get actual parameters because of some error, it would be better to throw an exception.

Reply:
It will fail even earlier with 

> Failed to connect to https://teamcity.jetbrains.com/guestAuth/app/rest/builds/id:699994/resulting-properties: 403  Responding with error, status code: 403 (Forbidden).